KarhooSDK/Api/Response/RefreshToken.swift

```diff
@@ -19,8 +19,11 @@ struct RefreshToken: KarhooCodableModel {
     }
 
     func toCredentials(withRefreshToken refreshToken: String?) -> Credentials {
-        return Credentials(accessToken: accessToken,
-                           expiresIn: TimeInterval(expiresIn),
-                           refreshToken: refreshToken)
+        return Credentials(
+            accessToken: accessToken,
+            expiresIn: TimeInterval(expiresIn),
+            refreshToken: refreshToken,
+            refreshTokenExpriresIn: TimeInterval(expiresIn)
```

Comment:
Should we use the same `expiresIn` value for refresh token? Shouldn't we have separate one for refresh token?

Reply:
I removed whole `RefreshToken.swift` file and `RefreshTokenMock.swift`. It looks for me like some old, unused filed. In every please in SDK we are using `AuthToken` struck.